data/spells/lib/spells.lua

```diff
@@ -348,3 +348,36 @@ function Creature:addPartyCondition(combat, variant, condition, baseMana)
 	return true
 end
 
+function Player:conjureItem(reagentId, conjureId, conjureCount, effect)
+	if not conjureCount and conjureId ~= 0 then
+		local itemType = ItemType(conjureId)
+		if itemType:getId() == 0 then
+			return false
+		end
+
+		local charges = itemType:getCharges()
+		if charges ~= 0 then
+			conjureCount = charges
+		end
+	end
+
+	if reagentId ~= 0 and not self:removeItem(reagentId, 1, -1) then
+		self:sendCancelMessage(RETURNVALUE_YOUNEEDAMAGICITEMTOCASTSPELL)
+		self:getPosition():sendMagicEffect(CONST_ME_POFF)
+		return false
+	end
+
+	local item = self:addItem(conjureId, conjureCount)
+	if not item then
+		self:sendCancelMessage(RETURNVALUE_NOTPOSSIBLE)
+		self:getPosition():sendMagicEffect(CONST_ME_POFF)
+		return false
+	end
+
+	if item:hasAttribute(ITEM_ATTRIBUTE_DURATION) then
+		item:decay()
+	end
+
+	self:getPosition():sendMagicEffect(effect)
```

Comment:
I still have not researched any effect, but I expect to soon, so only then I'll have a say if we would really need a default, but in any case I can add it if the same effect is being used by the majority of the scripts (after the research).

I'm still updating spells.xml, soon I'll mark this as concluded xD.